test.py

```diff
@@ -365,7 +365,7 @@ async def recycle_cluster(cluster: ScyllaCluster) -> None:
                which would delete the log file and directory - we might want to preserve
                these if it came from a failed test.
             """
-            await cluster.stop()
+            await cluster.stop_gracefully()
```

Comment:
We can't stop_gracefully without a timeout. It may hang and will hang the entire test suite. Failing tests should not have impact on nnon-failing ones. Please try to stop gracefully and revert to normal stop after a timeout.

Reply:
The timeout is in the stop_gracefully implementation...